packages/react-ui-ag/src/Modals/MoreOptionsModal.js

```diff
@@ -0,0 +1,195 @@
+
+    if (count > 1) {
+      count = this.formatNumberWithCommas(count)
+      propertyCount = `Show ${count} Properties`
+    } else if (count === 1) {
+      propertyCount = 'Show 1 Property'
+    }
+
+    return propertyCount
+  }
+
+  @autobind
+  getPropertyCount() {
+    const { filterTotal, total } = this.props
+    let count = (filterTotal !== null) ? filterTotal : total
+    let propertyCount = '0 Properties Found'
+
+    if (count > 1) {
+      count = this.formatNumberWithCommas(count)
+      propertyCount = `${count} Properties Found`
+    } else if (count === 1) {
+      propertyCount = '1 Property Found'
+    }
+
+    return propertyCount
+  }
+
+  @autobind
+  handleSubmit() {
+    const {
+      onSubmit,
+      onClose,
+    } = this.props
+
+    if (onSubmit) {
+      onSubmit()
+    }
+
+    if (onClose) {
+      onClose()
+    }
+
+    this.setState({ isOpen: false })
+  }
+
+  @autobind
+  handleCancel() {
+    const {
+      onClose,
+      restoreFilters,
+    } = this.props
+    restoreFilters()
+    onClose()
+  }
+
+  formatNumberWithCommas(number) {
+    return String(number).replace(/(.)(?=(\d{3})+$)/g, '$1,')
+  }
+
+  renderHeader() {
+    const {
+      onClose,
+      theme,
+    } = this.props
+
+    const propertyCount = this.getPropertyCount()
+
+    return (
+      <div className={theme.MoreOptionsModal_Header}>
+        <Text
+          className={theme.MoreOptionsModal_PropertyCount}
+        >
+          {propertyCount}
+        </Text>
+        <Button
+          className={theme.MoreOptionsModal_CloseButton}
+          onClick={() => onClose()}
+        >
+          Close
+        </Button>
+      </div>
+    )
+  }
+
+  renderFooter() {
+    const {
+      theme,
+      clearFilters,
+    } = this.props
+
+    const showPropertyButton = this.getShowPropertyButton()
+
+    return (
+      <div className={theme.MoreOptionsModal_Footer}>
+        <Button
+          className={theme.MoreOptionsModal_ResetAll}
+          onClick={() => clearFilters()}
+        >
+          Reset All
+        </Button>
+        <div className={theme.MoreOptionsModal_FooterButtons}>
+          <Button
+            className={theme.MoreOptionsModal_Cancel}
+            onClick={this.handleCancel}
+          >
+            Cancel
+          </Button>
+          <Button
+            className={theme.MoreOptionsModal_ShowProps}
+            onClick={this.handleSubmit}
+          >
+            {showPropertyButton}
+          </Button>
+        </div>
+      </div>
+    )
+  }
+
+  render() {
+    const {
+      theme,
+      moreOptionsForm,
+      onClose,
+      isOpen,
+    } = this.props
+
+    return (
+      <Modal
+        className={theme.MoreOptionsModal}
+        onClose={onClose}
+        isOpen={isOpen}
+      >
+        <div className={theme.MoreOptionsModal_InnerContainer}>
+          {this.renderHeader()}
+          <div className={theme.MoreOptionsModal_FiltersContainer}>
```

Comment:
Why do we need this? Can't the class be added to the `moreOptionsForm`?

Reply:
@tadjohnston yes it can be. will do